run-checks

```diff
@@ -106,6 +106,12 @@ if [ "$STATIC" = 1 ]; then
     for file in $(ls . | grep -v 'vendor\|po'); do
         ${GOBIN:-$GOPATH/bin}/misspell -error -i auther $file
     done
+
+    echo Checking for ineffective assignments
+    go get -u github.com/gordonklaus/ineffassign
+    for file in $(ls . | grep -v 'vendor\|po'); do
```

Comment:
ls . will just find one level of things, no? I used:

`find -name "*.go" -exec ineffassign {} \; | grep -v vendor`

Or am I missing something and is this thing doing it per directory?